corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -0,0 +1,64 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% requirejs_main 'app_manager/js/manage_releases' %}
+
+{% block page_content %}
+    {% initial_page_data 'enabled_app_releases' enabled_app_releases %}
+    {% registerurl 'deactivate_release_restriction' domain '---'%}
+    {% registerurl 'activate_release_restriction' domain '---'%}
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="panel panel-modern-gray panel-form-only">
+                <div class="panel-body">
+                    <form class="form-horizontal disable-on-submit" id="manage-app-releases" action=""
+                          method='post'>
+                        {% crispy manage_releases_form %}
+                    </form>
+                {% if enabled_app_releases %}
+                    <table id="managed-releases" class="table">
+                        <thead>
+                            <tr>
+                                <th>{% trans 'Application' %}</th>
+                                <th>{% trans 'Location' %}</th>
+                                <th>{% trans 'Version' %}</th>
+                                <th>{% trans 'Last Activity' %}</th>
+                                <th>{% trans 'Action' %}</th>
+                            </tr>
+                        </thead>
+                        <tbody data-bind="foreach: enabledAppReleases, visible: enabledAppReleases">
+                            <tr data-bind="attr: {id: domId}, css: {'bg-success': active() }">
+                                <td data-bind="text: app"></td>
+                                <td data-bind="text: location"></td>
+                                <td data-bind="text: version"></td>
+                                <td>
+                                    <div data-bind="visible: activatedOn">
+                                        {% trans 'Activated On : ' %}
+                                        <span data-bind="text: activatedOn"></span>
+                                    </div>
+                                    <div data-bind="visible: deactivatedOn">
+                                        {% trans 'Deactivated On : ' %}
+                                        <span data-bind="text: deactivatedOn"></span>
+                                    </div>
+                                </td>
+                                <td>
+                                    <a href="#" data-bind =
+                                    "click: toggleRestriction, text: actionText, visible: !ajaxInProgress()"></a>
+                                    <span class="spinner fa fa-spinner" data-bind="visible: ajaxInProgress"></span>
```

Comment:
I don't think the `spinner` class is doing anything. `fa-spin fa fa-refresh` is what we usually use for buttons that are working - the `fa-refresh` makes it the circular arrow image and the `fa-spin` makes it animate.

Reply:
`spinner` is used for handling visibility it during ajax request.
Are you suggesting to use a button instead of a link and then updating classes on it during ajax request?

Reply:
Seeing the following comment where you suggest updating this to a button.